bin/package-oss

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash -e
```

Comment:
Instead of creating a new script, can you update `bin/package` instead? Could also rename it `bin/package-electron`. All the code here should be for the OSS version so the "oss" itself is redundant (also curious what changes were made so I'd like to see the git diff)

Reply:
Will do!